tests_metricflow/snapshots/test_query_output.py/str/DuckDB/test_multiple_time_spines__query_output.txt

```diff
@@ -0,0 +1,41 @@
+test_name: test_multiple_time_spines
+test_filename: test_query_output.py
+---
+metric_time__martian_day    metric_time__hour    subdaily_join_to_time_spine_metric    subdaily_cumulative_window_metric
+--------------------------  -------------------  ------------------------------------  -----------------------------------
+2020-01-08T00:00:00         2020-01-01T01:00:00  1                                     1
+2020-01-08T00:00:00         2020-01-01T02:00:00  1                                     2
+2020-01-08T00:00:00         2020-01-01T03:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T04:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T05:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T06:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T07:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T08:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T09:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T10:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T11:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-01T12:00:00  1                                     3
+2020-01-08T00:00:00         2020-01-02T01:00:00  None                                  None
```

Comment:
These nulls are expected due to the the `join_to_timespine` metric. They span all the rows that we have in our hourly test time spine.

Reply:
Note: I'm planning to make some improvements to our time spine test data up the stack. Namely:
- Some of the feedback you gave me previously about `martian_day`
- Adding at least one more custom granularity (something fairly standard like `fiscal_quarter`)
- Adding more test data to our hourly time spine so that it at least spans more than one custom granularity period